_router_pon/free_iliad_F-MDCONU3A.md

```diff
@@ -0,0 +1,264 @@
+Type: FF Subtype FF Flags FF
+Stream: 4294967295 Revision: 4294967295
+Time: 65535-255-255 255:255:255Z
+
+App 1: pass
+2000 0001 v3.2.7 (Rel)
+Size: 319044 CRC: 0x2FAF887F
+Type: 02 Subtype 0C Flags 04
+Stream: 112 Revision: 131152
+Time: 2016-05-18 01:28:44Z
+
+App 2: pass
+2000 0001 v3.2.9 (Rel)
+Size: 319240 CRC: 0x3FBE2A30
+Type: 02 Subtype 0C Flags 04
+Stream: 116 Revision: 167733
+Time: 2019-03-13 01:47:37Z
+
+Diag: pass
+2000 0001 v3.2.7 (Rel)
+Size: 319044 CRC: 0x2FAF887F
+Type: 02 Subtype 0C Flags 04
+Stream: 112 Revision: 131152
+Time: 2016-05-18 01:28:44Z
+```
+
+`mem/rf [start address] [lenght]` reads bytes from the flash memory, wraps every 512 kB.
+
+## Firmware is interchangeable with
+
+## List of software versions
+
+## List of partitions
+
+The flash memory is not actually partitioned, upon reset the CPU loads from address 0 (reset vector) and jumps to another address ([page 74](http://me.bios.io/images/d/dd/ARCompactISA_ProgrammersReference.pdf#%5B%7B%22num%22%3A177%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C72%2C157%2C0%5D)). Each section ends with its CRC.
+
+| Section               | Start address    | End address      | Size           |
+|-----------------------|------------------|------------------|----------------|
+| Bootloader            | 0                | 42896/0xA790     | 42896/0xA790   |
+| App 0                 | ?                | ?                | ?              |
+| App 1                 | 1179687/0x120027 | 1498731/0x16DE6B | 319044/0x4DE44 |
+| App 2                 | 1703975/0x1A0027 | 2023215/0x1EDF2F | 319240/0x4DF08 |
+| Diag (copy of App 1?) | 2555943/0x270027 | 2874987/0x2BDE6B | 319044/0x4DE44 |
+
+(End address is non-inclusive)
+App 1 and App 2 sections are located at a distance of 512 kB (0x80000) from each other. This probably means that the CPU is capable of addressing only 512 kB of flash. It can be verified also by running the `mem/rf` command, which wraps every 512 kB.
+
+# Userful files and binaries
+
+## Useful files
+
+## Useful binaries
+
+# GPON ONU status
+
+## Get the operational status of the ONU
+
+## Get information of the OLT vendor
+
+## Querying a particular OMCI ME
+
+## Getting/Setting Speed LAN Mode
+
+# GPON/OMCI settings
+
+## Getting/Setting ONU GPON Serial Number
+
+## Getting/Setting ONU GPON PLOAM password
+
+## Getting/Setting ONU GPON LOID and LOID password
+
+## Getting/Setting OMCI software version (ME 7)
+
+## Getting/Setting OMCI hardware version (ME 256)
+
+## Getting/Setting OMCI vendor ID (ME 256)
+
+## Getting/Setting OMCI equipment ID (ME 257)
+
+# Advanced settings
+
+## Setting management MAC
+
+## Setting management IP
+
+## Rebooting the ONU
+
+## Creating a new rootfs
+
+## Flashing a new rootfs
+
+# SFP EEPROM settings
+
+## Reading all EEPROM
+
+# EEPROM
+
+There is an SFP plug on the UNI side with an embedded EEPROM.
+
+## EEPROM layout
+
+```
+00000000  03 04 00 00 00 00 00 00  00 00 00 00 0a 00 00 00  |................|
```

Comment:
please use a human-readable table, like table for eeprom https://hack-gpon.org/ont-huawei-ma5671a/#eeprom0-layout